packages/angular_devkit/build_angular/test/browser/rebuild_spec_large.ts

```diff
@@ -330,7 +324,7 @@ describe('Browser Builder rebuilds', () => {
     host.replaceInFile('./src/app/app.component.ts', `'app-root'`, `(() => 'app-root')()`);
 
     const overrides = { watch: true, aot: true };
-    const logger = new TestLogger('rebuild-aot-errors');
+    const logger = new TestLogger('rebuild-aot-errors', createConsoleLogger());
```

Comment:
Can you explain why the `createConsoleLogger` logger is needed here?

Reply:
It's what I use to see what's happening in a failing test that I don't expect to fail :D